NEWS.md

```diff
@@ -1,3 +1,7 @@
+## 12.3.0 IN-PROGRESS
+
+* implements normalization on ISBN index (CIRCSTORE-186)
```

Comment:
How would a client use this, is it an index to be used in a CQL query?

Reply:
Yup.  I'll replace this with the more verbose description in the jira issue-that should be clearer.